### ISSUE TITLE
ExposedDropdownMenu in Night mode shows incorrect text color

### ISSUE THREAD

**Issue by MariamSaab** [material-components/material-components-android#2045]
In this "@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense.ExposedDropdownMenu" in Night mode the text of dropdown items are black. The problem exists just in Night mode, in Day mode everything is ok.

This text of items of the dropdown when opened are shown over the "colorSurface", so they should be using the "colorOnSurface" for the text. In screenshot the "colorSurface is green. The text is unchangeable and it should work correctly without having to be customized. In night mode all other aspects work correctly. When I select the item it also works by showing white text. I would expect the behavior to be like the selected item color. 

In short the items in the dropdown should match the text color of the selected item. As you can also see in the screenshot I set the "colorOnSurface" to be white and it will also not change the text to white in the dropdown.

This problem happens on each device any API.

material version = '1.3.0-rc01'

This problem happens on each device. 

Can you give me the solution how to solve this issue please?

<img width="1019" alt="issuewithdropdown" src="https://user-images.githubusercontent.com/36329705/107073968-b7e54500-67f0-11eb-9cdf-94687ce8c5de.png">

**Comment by etwilliams** [comment]
I'm seeing this same issue on 1.3.0

**Comment by drchen** [comment]
Is this still an issue? 

The text color will be set to ?attr/textAppearanceBody1, which will by default use android:textColorPrimary as the text color.

Please check if you are overriding any of them.

**Comment by drchen** [comment]
I'll close the issue due to no further info available. Feel free to reopen it if it's still reproducible.

**Comment by markseon** [comment]
I ran into this issue when using a material theme created using the Figma Material Theme Builder plugin. The issue was resolved by using a theme generated by the web version of Material Theme Builder.

[material-theme.zip](https://github.com/user-attachments/files/20660889/material-theme.zip)
[material-theme-Figma.zip](https://github.com/user-attachments/files/20660890/material-theme-Figma.zip)
